code/modules/reagents/reagent_containers/food/snacks/meat.dm

```diff
@@ -125,6 +125,16 @@
 	reagents.add_reagent(NUTRIMENT, 3)
 	bitesize = 1
 
+/obj/item/weapon/reagent_containers/food/snacks/meat/rawchicken/raw_vox_chicken
+	name = "vox chicken meat"
+	desc = "Vox, man. No discussion."
+	icon_state = "raw_vox_chicken"
+
+/obj/item/weapon/reagent_containers/food/snacks/meat/rawchicken/raw_vox_chicken/New()
+	..()
+	reagents.add_reagent(NUTRIMENT, 3)
+	bitesize = 1
```

Comment:
and this one

Reply:
That's how the chicken meat and some other meats do it, but changed as well.